### PR TITLE
Use LinkedListMultiMap instead of HashMultiMap to preserve order of control plugins.

### PR DESCRIPTION
**Pull request by kevinsharpjr**

### CODE REVIEW

File: src/main/java/org/killbill/billing/client/KillBillClient.java [hunk 1]
```diff
@@ -34,6 +34,7 @@
 
 import javax.annotation.Nullable;
 
+import com.google.common.collect.*;
```
Comment (pierre) [inline src/main/java/org/killbill/billing/client/KillBillClient.java]: Let's avoid wildcard imports.

**Comment by pierre** [comment]
👍

File: src/main/java/org/killbill/billing/client/KillBillClient.java (hunk 1 above)
Comment (kevinsharpjr) [inline src/main/java/org/killbill/billing/client/KillBillClient.java]: Fixed.

**Comment by pierre** [comment]
Thanks!

**Pull request merged**